Modules/_ctypes/callbacks.c

```diff
@@ -378,10 +391,12 @@ CThunkObject *_ctypes_alloc_callback(PyObject *callable,
     }
 
     p->flags = flags;
+    PyObject **cnvs = PySequence_Fast_ITEMS(converters);
```

Comment:
Again, you can rely on the fact that it's a tuple. But please add a second assertion.